specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2016-05-01/BackupLocations.json

```diff
@@ -0,0 +1,315 @@
+                        "description": "ACCEPTED",
+                        "schema": {
+                            "$ref": "Backups.json#/definitions/Backup"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "BackupLocation": {
+            "description": "Information about the backup location.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "description": "Properties of a backup location.",
+                    "$ref": "#/definitions/BackupLocationModel",
+                    "x-ms-client-flatten": true
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "Backup.json#/definitions/Resource"
+                }
+            ]
+        },
+        "BackupLocationModel": {
+            "type": "object",
+            "description": "Properties of a backup location.",
+            "properties": {
+                "externalStoreDefault": {
+                    "description": "Information about an external storage location.",
+                    "$ref": "#/definitions/ExternalStore",
+                    "x-ms-client-flatten": true
+                }
+            }
+        },
+        "ExternalStore": {
+            "type": "object",
+            "description": "Information about an external storage location.",
+            "properties": {
+                "path": {
+                    "description": "Path to the update location",
+                    "type": "string"
+                },
+                "userName": {
+                    "description": "Username to access the location.",
+                    "type": "string"
+                },
+                "password": {
+                    "description": "Password to access the location.",
+                    "type": "string"
+                },
+                "encryptionKeyBase64": {
+                    "description": "Encryption key.",
+                    "type": "string"
+                },
+                "backupFrequencyInHours": {
+                    "description": "The interval, in hours, for the frequency that the scheduler takes a backup.",
+                    "type": "string"
+                },
+                "availableCapacity": {
+                    "description": "Free space at the backup location.",
+                    "type": "string"
+                },
+                "isBackupSchedulerEnabled": {
+                    "description": "True if the backup scheduler is enabled.",
+                    "type": "boolean"
+                },
+                "nextBackupTime": {
+                    "description": "The scheduled time of the next backup.",
+                    "type": "string"
+                },
+                "lastBackupTime": {
+                    "description": "Time of backup.",
+                    "type": "string"
```

Comment:
Should we use date type here?

```json
        "lastBackupTime": {
          "type": "string",
          "format": "date-time",
          "description": "Time of backup."
        }
```

Reply:
Leaving as a string.  I have not seen any returned value which I can verify is a date-time.  String is more general and if result is not a date-time we will get a deserialization error.